tsconfig.json

```diff
@@ -14,17 +14,7 @@
     "esModuleInterop": true,
     "noImplicitOverride": true,
     "noImplicitAny": false,
-    "module": "esnext",
-    "moduleResolution": "node",
-    "outFile": "dist/types/index.d.ts",
-    "skipLibCheck": true,
-    "target": "ES2015",
-    "paths": {
-      // matches the alias in webpack config, so that types for those imports are visible.
-      "@/*": [
-        "src/*"
-      ]
-    }
+    "outFile": "dist/types/index.d.ts"
```

Comment:
look at all the deletion - wunderbar!

Reply:
right! we're not transpiling typescript, so i was able to remove all of the options that are solely meant for that